### PR TITLE
Adding a blog post about ELevate Updates

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

**Comment by sboldyreva** [comment]
@bennyvasquez

File: content/blog/2024-01-30-elevate-updates.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## ELevate now supports EPEL and more
+
+ELevate, a fork of Red Hat's Leapp project, supports migrations between CentOS 7 and Enterprise Linux (EL) 8, and between EL8 and EL9 systems. It has helped more than 50,000 systems upgrade in place, allowing our global community to easily keep their infrastructure updated and more secure. The AlmaLinux community is constantly working on improving the project.
+
+Originally, Red Hat’s Leapp only offered support for Red Hat Enterprise Linux and its official repositories. However, ELevate now offers 3rd party repository support. Specifically, we recently announced a testing version of ELevate that [included EPEL support](https://almalinux.org/blog/2023-12-05-announcing-epel-support-in-elevate/) for migrations from CentOS 7 to AlmaLinux 8. Today, this feature is available in the **stable** version of ELevate.
+
+In addition, we have expanded beyond just EPEL support. Migration with additional repositories like Imunify, KernelCare, MariaDB, nginx, and PostgreSQL is available for all supported systems including AlmaLinux, CentOS, EuroLinux, Oracle Linux, and Rocky Linux.
```
Comment (codyro) [inline content/blog/2024-01-30-elevate-updates.md:24]: Is this the proper place for Imunify & KernelCare repository mentions?

File: content/blog/2024-01-30-elevate-updates.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: "ELevate Project Updates"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: 2024-01-30
```
Comment (bennyvasquez) [inline content/blog/2024-01-30-elevate-updates.md]:
```suggestion
date: 2024-01-31
```

**Review by bennyvasquez — APPROVED** [review]
With this one change, it's ready!

File: content/blog/2024-01-30-elevate-updates.md (hunk 1 above)
Comment (bennyvasquez) [reply content/blog/2024-01-30-elevate-updates.md]: @codyro I think this is the full list of third party repos that we've historically interacted with, so it seems like the right place for it to me. If it's 'cause they're commercial, that makes sense. I think it's okay to leave them for now, but we should talk about some guidelines around when we do or don't mention commercial stuff in blog posts like this.

File: content/blog/2024-01-31-elevate-updates.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+title: "ELevate Project Updates"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: 2024-01-30
```
Comment (bennyvasquez) [inline content/blog/2024-01-31-elevate-updates.md:8]:
```suggestion
date: 2024-01-31
```

**Pull request merged**